session/src/main/scala/akka/cassandra/session/scaladsl/CassandraSessionRegistry.scala

```diff
@@ -81,7 +82,20 @@ final class CassandraSessionRegistry(system: ExtendedActorSystem) extends Extens
       executionContext: ExecutionContext): CassandraSession = {
```

Comment:
there is also a "strange" comment above "Note that the session must not be stopped manually...", which I don't think is correct.